pkgs/applications/misc/megasync/default.nix

```diff
@@ -25,16 +25,18 @@
 , unzip
 , wget
 }:
-
+let
+  srcFlavor = "Linux";
```

Comment:
Why are we introducing this? It can't be overwritten right now.

Reply:
I was trying out "Win" flavor and I found it convenient. Nevertheless, using other flavors probably does not make sense as long as "Linux" is available for particular version.
I've just removed this change.